README.md

```diff
@@ -5,7 +5,7 @@
 > _Dis moi_ means _Tell me_ in :fr: French.
 
 **Dismoi** is a web extension that allows anyone to post information directly on any web page you browse.
-If you follow a contributor, his information messages are displayed at the time you visit the pages he has commented on.
+If you follow an informer, his messages are displayed at the time you visit the pages he has commented on.
```

Comment:
It looks like `informant` is the most common term in english :

https://www.linguee.fr/francais-anglais/search?source=auto&query=informateur

Reply:
Hum, this is also interesting :
https://wikidiff.com/informer/informant

Reply:
Nice ! Totally convinced !